### ISSUE TITLE
Start mneu Windows 10 multiple extraneuous Terminal -Entries

### ISSUE THREAD

**Issue by theking2** [valinet/ExplorerPatcher#1246]
With the Start-Menu in Win10 mode multiple extraneous PowerShell/Terminal entries appear. 
![image](https://user-images.githubusercontent.com/1612152/229429026-f72befe6-caae-4bac-966e-09081bc4a786.png)

Probably due to updates of Powershell.
Switching to Win11 start and back to Win10 solves this.

**Comment by MikeRotch76** [comment]
I have this issue as well.  When I uninstall EP this is what my win+X menu looks like

![image](https://github.com/valinet/ExplorerPatcher/assets/70385220/8dbc337f-8507-4ccf-9c61-ab6a7695551f)

This is what it looks like with EP installed.

![image](https://github.com/valinet/ExplorerPatcher/assets/70385220/ddcb75e8-ddac-4d24-8e32-2ac12cc48c55)

I am on Windows 11 Home 22621.1848

**Comment by pyrates999** [comment]
Might be because windows 10 had the option Windows PowerShell (Admin), while windows 11 has Terminal (Admin).  Try setting it back to windows 11, restart file explorer, and see if Windows PowerShell (Admin) is still there.

**Comment by gigabet** [comment]
I use 🪟+X A a lot in my workflow, and the multiple terminal entries on the W10 taskbar make that shortcut not work. Which makes it unsuable for me.

**Comment by MikeRotch76** [comment]
> Might be because windows 10 had the option Windows PowerShell (Admin), while windows 11 has Terminal (Admin). Try setting it back to windows 11, restart file explorer, and see if Windows PowerShell (Admin) is still there.

Yes this works, but it defeats the purpose of me using EP, which is to give me the Windows 10 taskbar.

**Comment by pyrates999** [comment]
> > Might be because windows 10 had the option Windows PowerShell (Admin), while windows 11 has Terminal (Admin). Try setting it back to windows 11, restart file explorer, and see if Windows PowerShell (Admin) is still there.
> 
> Yes this works, but it defeats the purpose of me using EP, which is to give me the Windows 10 taskbar.

Then you've confirmed this is a bug that should be fixed with explorer patcher.  Now you wait to see for it to get fixed.

**Comment by meatcar** [comment]
I got the <kbd>Win+X</kbd><kbd>I</kbd>/<kbd>A</kbd> shortcuts working using the windows 10 taskbar in windows 11 in a roundabout hacky way. You have to rename the Powershell entries to "pwsh" using a win+x editing tool like [winaero's](https://winaero.com/download-winx-menu-editor-allows-you-to-add-or-remove-winx-menu-items/). 

The menu now looks like this, note the underlines:

![image](https://github.com/valinet/ExplorerPatcher/assets/191622/c2e1ffd2-f634-44b1-805a-0264a4939080)

Seems like the "Terminal" entries are added by the win10 taskbar, and are hardcoded, not respecting the `AppData\Local\Microsoft\Windows\WinX` shortcut names.
